packages/server/src/integrations/utils.ts

```diff
@@ -328,3 +328,22 @@ export function finaliseExternalTables(
     .reduce((r, [k, v]) => ({ ...r, [k]: v }), {})
   return { tables: finalTables, errors }
 }
+
+/**
+ * Checks if the provided input is an object, but specifically not a date type object.
+ * Used during coercion of types and relationship handling, dates are considered valid
+ * and can be used as a display field, but objects and arrays cannot.
+ * @param test
+ */
+export function getPrimaryDisplay(test: any): string | undefined {
```

Comment:
This being in the utils it looks like it's going to be reused. But with the description/name/params it's not really clear what is it used. Could we make the function name and params more verbose?

Reply:
yeap - will do!